### ISSUE TITLE
Question on ASM1117

### ISSUE THREAD

**Issue by petanque** [martinloren/HScope#4]
Hi, I wonder why you use IC AMS1117-1.2v in 5v Best Module.
The output pin is not used, is 1.2v important or can it be replaced with a 1.8v or 3.3v?

**Comment by martinloren** [comment]
Ive tested ASM1117-ADJ version and it work as well. This configuration is not documented and provide VCC/2 required to shift the input signal in the middle of the ADC range. In this way you can have a dual input without having a dual voltage ADC stage normally required for these devices. This allow the less number of components possible.

**Comment by MikkelSnitker** [comment]
Hi MartinLoren.

So the output of the ASM1117 is not connected to anything? 

Since my electronics pusher doesn't have the ASM1117, I was wondering if you think the TS1117CW-ADJ  [(http://www.farnell.com/datasheets/50383.pdf)](http://www.farnell.com/datasheets/50383.pdf) would do as well?

**Comment by martinloren** [comment]
Yes, output not connected. For other components you should test. I've tested also another small version of ASM1117 and it is not working, it is not the same circuit.

**Comment by mmghv** [comment]
I honestly don't know how it's working and I need help, I don't have access to an AMS1117 but I have other regulators, does it have to be 1.2v or the voltage doesn't matter here since the output is not used, how can I test to see if my regulator is working as expected?

I'm very confused about the use of this AMS1117 in the circuit and I never seen it being used like this before.
